### ISSUE TITLE
Latest carbon v9 breaks DataTableSkeleton

### ISSUE THREAD

**Issue by nicoxxie** [carbon-design-system/carbon#2859]
## What package(s) are you using?

- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.
After updating to the latest carbon 9.91.2, DataTableSkeleton is not shown.  Verified that  9.91.1 does not work also.   9.91.0 works.

> Is this issue related to a specific component?
DataTableSkeleton

> What did you expect to happen? What happened instead? What would you like to
> see changed?
Loading skeleton is shown

> What browser are you working in?
Chrome Version 74.0.3729.169 (Official Build) (64-bit)

> What version of the Carbon Design System are you using?
9.91.2

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?
IBM cloud

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

This is the code we use
```
{this.props.loading ? <DataTableSkeleton rowCount={1} /> : <Table .../>}
```

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components/tree/master/examples/codesandbox
https://codesandbox.io/s/codesandbox-99gxn

## Additional information

- Screenshots or code
- Notes

![May-28-2019 14-00-40](https://user-images.githubusercontent.com/17915105/58505007-8f6b0380-8151-11e9-9d17-d4f999f91979.gif)

**Comment by nicoxxie** [comment]
No change of code using v9.91.0
![May-28-2019 14-15-24](https://user-images.githubusercontent.com/17915105/58505656-15d41500-8153-11e9-8107-88d5c251ff67.gif)

**Comment by emyarod** [comment]
the codesandbox doesn't seem to display anything even if I roll the dependency versions back in the sandbox

still looking into this but in any case I believe this regression is surfacing because `carbon-components-react`'s vanilla dependency version just jumped from `9.90.9` to `9.91.2`, so none of the vanilla releases in between were being consumed by `carbon-components-react`. my guess is the problem began somewhere in the intermediate releases

**Comment by emyarod** [comment]
looks like it was removed when v9 code was pruned and data table v2 was renamed to data table in v10

will open a PR to resolve this but in the meantime a temporary fix would be to add both classes to your skeleton table e.g. `<table class="bx--skeleton bx--data-table-v2 bx--data-table">`

**Comment by asudoh** [comment]
@emyarod Do you mean data table class change happened in `v9` codebase (non-roped-off code)?

**Comment by emyarod** [comment]
@asudoh seems so, since the skeletons are not styled unless the `bx--data-table` class is applied to the table

**Comment by asudoh** [comment]
I see, if you can spot what commit made such change it'll be great, to ensure we do the right thing to fix the problem. Thanks!

**Comment by nicoxxie** [comment]
Looks like the fix went in, if someone can push a new release that would be great, thank you

**Comment by nicoxxie** [comment]
verified fixed in 9.91.3
